packages/ui/src/documents/CreateDocument.tsx

```diff
@@ -86,23 +80,18 @@ export const CreateDocument: FunctionComponent<Props> = (props) => {
 
   const createDocument = async (document: Document) => {
     setState({
-      loadingMessage: 'Saving document',
       defaultDocument: document,
     });
 
     try {
-      const doc = (await httpClient.documents.create(document)).data;
-      push(documentRoutes.view.replace(':id', doc._id));
-
+      push(documentRoutes.index);
```

Comment:
Seems we are performing this `push` two times (also on line 100). Is that correct?

Reply:
this one is in the createDocument() method, the other one is in the onCancel() method